charts/redpanda/values.yaml

```diff
@@ -533,21 +533,17 @@ statefulset:
   securityContext:
     fsGroup: 101
     runAsUser: 101
+    fsGroupChangePolicy: OnRootMismatch
   initContainers:
     tuning:
       resources: {}
-    setDatadirOwnership:
-      resources: {}
     setTieredStorageCacheDirOwnership:
       resources: {}
     configurator:
       resources: {}
   initContainerImage:
     repository: busybox
     tag: latest
-  # in environments where root is not allowed, you cannot change the ownership of files and directories
-  # set this to skip this step
-  skipChown: false
```

Comment:
Is this `skipChown` part of `values.schema.json`? The helm upgrade with —reuse-values will brake or will just left the unused properties?

Reply:
Thank you for bringing that up! I meant to delete this from the schema, so thank you for bringing that up.

 So using --reuse-values will not cause an issue as the value and checking the values file after upgrading to this chart

```
  securityContext:
    fsGroup: 101
    fsGroupChangePolicy: OnRootMismatch
    runAsUser: 101
  skipChown: false
```

```
redpanda    	redpanda    	2       	2023-03-07 10:13:54.993363145 -0500 EST	deployed	redpanda-2.13.1     	v22.3.13   
```